src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+body {
+  margin: 0;
+  text-transform: uppercase;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+a {
+  display: inline-block;
+  padding: 2em 0;
+  cursor: pointer;
+  line-height: 1em;
+}
+
+ul {
+  list-style: none;
+}
+
+.logo__link {
+  font-size: 0;
+  display: flex;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  height: 60px;
```

Comment:
better to avoid height hardcoding for containers, let the content dictate container height